### ISSUE TITLE
login-cookie no longer seems to work (logs in for a minute then logs out)

### ISSUE THREAD

**Issue by jejb** [mautrix/googlechat#98]
I recently got logged out of my google chat account and tried to re-establish the login again, but it works for a few minutes and then logs out again.  This is a copy of the most recent chat:

Google Chat bridge bot
Connected to Google Chat
Successfully logged in as Jej B <[jejbgo@googlemail.com](mailto:jejbgo@googlemail.com)> (106426395028955095070)

Google Chat bridge bot
Logged out from Google account

The timestamps on the two messages are 10:08 and 10:15 so it stayed logged in for all of 7 minutes.

I think this means google has done an update to the authentication mechanism googlechat doesn't have

**Comment by tulir** [comment]
Did you close the private browser window after logging in (and preferably before or immediately after logging into the bridge)?

**Comment by jejb** [comment]
Yes, I did.  I just tried it again with a private browser window, this time being very careful to ensure I didn't have any others open and the same thing happened.  However, when I tried it and then killed the entire browser it didn't log me out (well 40 minutes and counting) so google may have found a way to get the browser to help with eliminating the refresh token.
